src/features/user/Profile/components/MyTrees/MyTrees.tsx

```diff
@@ -85,6 +85,20 @@ export default function MyTrees({ profile, authenticatedType, token }: Props) {
                 ? t('me:myForest')
                 : t('me:nameForest', { name: profile.displayName })}
             </div>
+            <div className={styles.donatedTreesContainer}>
+              <div>
+                <TreesIcon />
+              </div>
+              <p className={styles.donatedTrees}> {t('me:donatedTrees')} </p>
+              <div className={styles.registeredTreesContainer}>
+                <div style={{ marginLeft: '2px' }}>
+                  <TreesIcon color="#3D67B1" />
+                </div>
+                <p className={styles.registeredTrees}>
+                  {t('me:registeredTrees')}
+                </p>
+              </div>
+            </div>
```

Comment:
@sunilsabatp This can be structured a little better.

1. Logically, `donatedTreesContainer` and `registeredTreesContainer` are at the same level of hierarchy. But in this code, `registeredTreesContainer` is a child of `donatedTreesContainer`. 
    a. I'd suggest keeping a parent container element for both donatedTreesContainer and registeredTreesContainer. 
    b. You could keep a common structure for both donatedTreesContainer and registeredTreesContainer - each containing the icon and the label. 
2. Perhaps the naming could be better. For example, you could name the parent container element something like myTreesLegend.
3. Avoid inline styles like marginLeft: 2px. As you're using flexbox for the parent element, adjust spacing between donatedTreesContainer and registeredTreesContainer using the `gap` properties - you do not need to specify a px value.
4. Try to use common styles for donatedTreesContainer and registeredTreesContainer - the only thing that really differs is the colour.